packages/ruby_debug.rb

```diff
@@ -0,0 +1,63 @@
+# Adapted from Arch Linux ruby-rubocop PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=ruby-rubocop
+
+require 'package'
+
+class Ruby_debug < Package
+  description 'Ruby debug.rb'
```

Comment:
Better description?  Not really `debug.rb` but `ruby_debug.rb`.

Reply:
"Ruby\'s debug.rb" ?

Reply:
The official description?  https://rubygems.org/gems/debug/

Reply:
By the way, what would we call this if it was packaged, ruby_ruby_debug? https://rubygems.org/gems/ruby-debug/